go/webdriver/webdriver.go

```diff
@@ -70,6 +129,10 @@ type WebDriver interface {
 	Capabilities() map[string]interface{}
 	// Screenshot takes a screenshot of the current browser window.
 	Screenshot(context.Context) (image.Image, error)
+	// KeyDown performs key presses to the active element.
+	KeyDown(ctx context.Context, keys string) error
+	// KeyUp releases depressed keys.
```

Comment:
I think it's better to improve the comments to make it clear if the semantics of the "keys" parameter means pressing/releasing the keys at the same time or one by one.

Reply:
Ack. I'm mirroring an internal contribution. Let's follow up with a separate PR (discussed in 1:1).